securedrop/source_app/main.py

```diff
@@ -201,7 +203,9 @@ def login():
         if form.validate_on_submit():
             codename = request.form['codename'].strip()
             if valid_codename(codename):
-                session.update(codename=codename, logged_in=True)
+                session.update(codename=codename,
+                               logged_in=True,
+                               new_user=False)
```

Comment:
I think we even could be lazier here and not set this to `False` ever and just let it be absent in the session. This relates to my other comment.